inc/ADC.h

```diff
@@ -1,14 +1,17 @@
 #ifndef ADC_H_
 #define ADC_H_
 
-extern uint16_t ADC_Values[17][5];
+#define sensor_quantity 17
+#define ammount_of_recorded_values 5
+
+uint16_t ADC_Values[sensor_quantity][ammount_of_recorded_values];
```

Comment:
It looks like you are declaring ADC_Values in both the ADC.c and ADC.h. You shouldn't declare variables in the header file. If you would like other files to have access to ADC_Values, declare it in ADC.c and include in ADC.h with the "extern" in front of it. "extern uint16_t ADC_Values[sensor_quantity][amount_of_recorded_values].